docs/reference/components.cue

```diff
@@ -442,5 +442,9 @@ components: {
 				}
 			}
 		}
+
+		telemetry: metrics: {
```

Comment:
I'd go ahead and all of the metrics definitions here. Ex:

```
_vector_events_processed_total: ...
_vector_processed_bytes_total: ...
```

And drop the top-level `_metrics` key.